docs/technical/mock-classifier-technical-architecture.md

```diff
@@ -0,0 +1,110 @@
+# Mock Classifier Technical Architecture
+
+This document outlines the technical design for the mock classifier component.
+
+## Introduction
+
+Classifier has a single responsibility which is to determine which Ministry can answer the query from a user via a bot.
+
+The Classifier takes raw messages from user's as an input and returns the Ministry which can answer these message as an output.
+
+The design and build of the real Classifier will be undertaken in parallel to the main project by a separate team. For the purposes of the project, we will create a mock Classifier which takes the same inputs as the real service and produces the same outputs, but has hard-coded resolutions.
+
+The mocked Classifier Service will offer two main features:
+
+- For demo, we will match hard coded strings (in English) to Ministries. For example "Thanks. How do I register my daughter at a local school?" would return "education" as the Ministry
+- For testing and broader scenarios, we will also have a token format where we can pass in the exact Ministriy we want to return
+
+Once the appropriate Ministry has been resolved by Classifier, DMR will consult CentOps to map the Ministry to a specific bot uri. It is not Classifier's job to resolve text to bots or institutions, only ministries.
```

Comment:
Once the appropriate Ministry has been resolved by Classifier, DMR, been provided with appropriate information by CentOps, maps the Ministry to a specific bot uri. It is not Classifier's job to resolve text to bots or institutions, only ministries.

Reply:
Suggested change has been applied, thanks